### ISSUE TITLE
Unicode issue

### ISSUE THREAD

**Issue by bkircher** [otris/vscode-janus-debug#16]
![cap-2](https://cloud.githubusercontent.com/assets/536464/24200322/1a5aba80-0f0d-11e7-9cd4-0f5abc59ae39.png)

The script appears to be valid UTF-8.

**Comment by bkircher** [comment]
Issue still not fixed despite the changes in commit 3300e20674da1ac07b18fa43c37dbb54e3219a50. The commit destroyed the display of the variable values in the "Locals" windows on the left. I added f237ec71875d1f435b8e3b9aa548612c100676e0 to fix that.

I looked at the documentation of the new `utf8` module and I am not sure what this is supposed to do:


```ts
/**
 * Decodes any given UTF-8-encoded string (byteString) as UTF-8, and returns the
 * UTF-8-decoded version of the string. It throws an error when malformed UTF-8
 * is detected.
 */
function decode(byteString: string): string;

```
So this does what exactly? Nothing?

Anyways, following shows the current state now. The uploaded script is encoded in ISO-8859-15 and the "Locals" is still garbled up.

![cap-3](https://cloud.githubusercontent.com/assets/536464/26722345/cbbeaf2e-478f-11e7-9942-0fefd9322629.png)
